chia/full_node/full_node.py

```diff
@@ -918,7 +928,21 @@ async def on_connect(self, connection: WSChiaConnection) -> None:
             synced = await self.synced()
             peak_height = self.blockchain.get_peak_height()
             if synced and peak_height is not None:
-                my_filter = self.mempool_manager.get_filter()
+                # any invalid transactions we've seen recently, we don't need
+                # to see again, so add those to the filter as well
+
+                # first remove transactions that are older than 2 minutes
```

Comment:
can you explain the 2 minutes aspect of this here

Reply:
this was an arbitrary timeout. it might make sense to make it longer. It seemed unnecessary to keep old transactions around indefinitely, that's all.

Reply:
I bumped it to 10 minutes